tests/ui/error-codes/E0583.stderr

```diff
@@ -5,6 +5,7 @@ LL | mod module_that_doesnt_exist;
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = help: to create the module `module_that_doesnt_exist`, create file "$DIR/module_that_doesnt_exist.rs" or "$DIR/module_that_doesnt_exist/mod.rs"
+   = note: if there is a `mod module_that_doesnt_exist { ... }` elsewhere in the crate already, import it with `use crate::...` instead
```

Comment:
there might be a `mod module_that_doesnt_exist;` instead, which may look different enough to be confusing. could we somehow also get that into the message somehow?
maybe just drop the braces and just use `mod module_that_doesnt_exist`, which is deliberately ambiguous?